src/index.html

```diff
@@ -13,5 +13,166 @@
   <body>
     <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    onsubmit="onSubmit()">
+
+      <!-- personal -->
+      <fieldset class="fieldset">
+        <legend> Personal information</legend>
+        <label class="form-field" for="surname">
+          Surname:
+          <input type="text"
+          autocomplete="false"
+          name="surname"
+          id="surname"
+          minlength="3"
+          maxlength="10">
```

Comment:
```suggestion
          <input
            type="text"
            autocomplete="false"
            name="surname"
            id="surname"
            minlength="3"
            maxlength="10"
          >
```
Add indentations. Closing bracket should be on the newline